YouTrackAutomation/Functions/Get-YTProject.ps1

```diff
@@ -47,7 +47,10 @@ function Get-YTProject
         [String] $ShortName,
 
         # Additional fields to include in the response.
-        [String[]] $AdditionalField
+        [String[]] $AdditionalField,
+
+        # Maximum number of results to return. Api end point defaults to top 42. Defaulted to 300 here.

```

Comment:
We should leave this blank so that we return the default number of results that the API returns. and only include the `top` query string parameter if this parameter is given. Document in the function's description that the API currently returns 42 results at a time.